packages/mssql/src/Adaptor.js

```diff
@@ -265,7 +254,9 @@ export function findValue(filter) {
     let conditionsArray = [];
     for (let key in whereData)
       conditionsArray.push(
-        `${key} ${operatorData ? operatorData[key] : '='} '${whereData[key]}'`
+        `${key} ${operatorData ? operatorData[key] : '='} '${escape(
```

Comment:
Because this is a very dense template, and because the escaping is very important, can we break out the escaped value into its own variable?

Reply:
Added `escapedValue` variable